openslides_backend/action/actions/meeting/update.py

```diff
@@ -114,6 +114,7 @@
     "motion_poll_default_backend",
     "users_enable_presence_view",
     "users_enable_vote_weight",
+    "users_enable_vote_delegations",
```

Comment:
Please add the new field to the wiki on action meeting.update

Reply:
Add the field to the action wiki